Source/cmake/GStreamerChecks.cmake

```
@@ -60,14 +60,15 @@ endif ()
 if (ENABLE_MEDIA_STREAM AND ENABLE_WEB_RTC)
     if (USE_GSTREAMER_WEBRTC)
         SET_AND_EXPOSE_TO_BUILD(USE_LIBWEBRTC FALSE)
-
-        find_package(OpenSSL)
-        if (NOT OPENSSL_FOUND OR OPENSSL_VERSION VERSION_LESS "3.0.0")
-            message(FATAL_ERROR "OpenSSL 3 is needed for USE_GSTREAMER_WEBRTC.")
-        endif ()
     else ()
         SET_AND_EXPOSE_TO_BUILD(USE_LIBWEBRTC TRUE)
     endif ()
+
+    # OpenSSL is required for SFrame encryption support.
+    find_package(OpenSSL)
+    if (NOT OPENSSL_FOUND OR OPENSSL_VERSION VERSION_LESS "3.0.0")
+        message(FATAL_ERROR "OpenSSL 3 is needed for SFrame encryption support.")
```

What's important here is to say what build option to turn off if you don't have OpenSSL 3 available, not to say what precisely it's being used for:

```suggestion
        message(FATAL_ERROR "OpenSSL 3 is needed for ENABLE_WEB_RTC.")
```